src/main/java/com/denizenscript/depenizen/bukkit/properties/breweryx/BreweryPlayerExtensions.java

```diff
@@ -0,0 +1,55 @@
+package com.denizenscript.depenizen.bukkit.properties.breweryx;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.dre.brewery.BPlayer;
+
+public class BreweryPlayerExtensions {
+
+    public static void register() {
+        // <--[tag]
+        // @attribute <PlayerTag.brewery_drunkenness>
+        // @returns ElementTag
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the drunkenness of the brewery player or null if Brewery has no data on the player.
```

Comment:
`if Brewery has no data on the player.` can probably be simplified to `if the player isn't drunk`? or is brewery's handling more complicated than that?